inference/models/florence2/florence2.py

```diff
@@ -0,0 +1,16 @@
+import torch
+from transformers import AutoModelForCausalLM
+
+from inference.models.transformers import LoRATransformerModel, TransformerModel
+
+
+class Florence2(TransformerModel):
+    hf_args = {"trust_remote_code": True}
```

Comment:
Looks interesting, does this imply remote code will be executed at some point?

Reply:
Yeah, code is downloaded from the `base_model` repo. Maybe we should turn that off and host florence2 ourselves?